packages/generate/src/edgeql-js.ts

```diff
@@ -187,7 +187,13 @@ export async function generateQueryBuilder(params: {
       .then((content) => content)
       .catch(() => "");
 
-    const newContents = headerComment + f.content;
+    let newContents = headerComment + f.content;
+
+    // in server versions >=6 cal, fts, math and pg are moved inside std module
+    if (f.path.split(".")[0] === "range" && version.major > 5) {
+      newContents = newContents.replace("modules/cal", "modules/std/cal");
```

Comment:
I guess this is unlikely to accidentally replace the wrong thing in the file, but we should make this generate time replacement more explicit in the syntax files? Maybe by re-naming the mock modules dir we import from to something like `MOCK_MODULES`, then doing replacement on all modules imports that match the pattern?

Reply:
That seems like a good idea in general. I think we need to add some additional code to ensure that we don't copy the `MOCK_MODULES` directory at generate time because I _think_ maybe the whole syntax dir is copied verbatim and then generation replaces stuff?

I don't think implementing this should be blocking though, we've already decided these fake modules existing is fine so this isn't any _more_ confusing than what we had before 😅

Reply:
yes, I think this is how it works. I didn't update the PR with MOCK_MODULES rename and update...
if you think the rest is enough we can push this and if needed I can add further updates